copy.client.ts

```diff
@@ -3,7 +3,7 @@ function findParent(el, find) {
     if (find(el)) {
       return el;
     }
-  } while (el = el.parentElement);
+  } while ((el = el.parentElement));
```

Comment:
is this some weird formatting rule?

Reply:
Now that I'm looking at this...I'm wondering if we should actually just rewrite this whole file. Seems odd to add the listener to the whole document and do this much DOM crawling. (I wasn't concerned with the JS at the time; I was just working on the CSS.) What do you think @thisisjofrank?

Reply:
Never mind; I went ahead and pulled the trigger on a rewrite, which I think makes much more sense (and is probably more performant) anyway. I also modified the styling a little bit.